tests/end-to-end/no-log-cc-override/no-log-no-cc.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+phpunit -c _files/phpunit.xml --no-logging --log-junit php://stdout _files/NoLogNoCcTest.php
+--FILE--
+<?php declare(strict_types=1);
+$arguments = [
+    '--configuration',
+    \realpath(__DIR__ . '/_files/phpunit.xml'),
+    '--no-logging',
+    '--log-junit',
+    'php://stdout',
+//    '--no-coverage',
```

Comment:
At start I redacted the test also with `--no-coverage` coverage, but was unable to load a driver within the PHPT test.

Any guess on how to test this properly?